packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -802,4 +860,4 @@ class _DragAvatar<T extends Object> extends Drag {
     }
     return Offset(0.0, offset.dy);
   }
-}
+}
```

Comment:
Files should end with a newline character
```suggestion
}

```